docs/common-errors.md

```
@@ -220,6 +220,23 @@ If you did not download precompiled boost, locate your Boost folder
 1. Run the Bootstrap.bat file
 1. Run the b2.exe file 
 
+---------------------------------------------------------
+
+#### ACE00068
+I can't generate my CMake files on Ubuntu 26.04, I get:
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)